blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java

```diff
@@ -61,13 +64,32 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
 
         Tracker.getInstance().track(request);
 
+        requestCount.inc();
+
+        if ( !mediaTypeChecker.isContentTypeValid(request.headers()) ) {
```

Comment:
May be we can create these as methods in DefaultHandler if there are common across handlers.